<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GL Scratchpad</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="ApplicationFrame" name="applicationFrameWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>251</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Program"/>
    <addaction name="actionClose_Program"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionShowCameraView"/>
   </widget>
   <widget class="QMenu" name="menuProgram">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="actionShowProgramView"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCamera"/>
   <addaction name="menuProgram"/>
  </widget>
  <action name="actionOpen_Program">
   <property name="text">
    <string>Open Program</string>
   </property>
  </action>
  <action name="actionClose_Program">
   <property name="text">
    <string>Close Program</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionShowCameraView">
   <property name="text">
    <string>Show View</string>
   </property>
   <property name="toolTip">
    <string>Show Camera View</string>
   </property>
  </action>
  <action name="actionShowProgramView">
   <property name="text">
    <string>Show View</string>
   </property>
   <property name="toolTip">
    <string>Show Program View</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ApplicationFrame</class>
   <extends>QWidget</extends>
   <header>applicationframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionShowCameraView</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>ShowCameraView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_Program</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>OpenProgram()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose_Program</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>CloseCurrentProgram()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowProgramView</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>ShowProgramView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OpenProgram()</slot>
  <slot>ShowCameraView()</slot>
  <slot>CloseCurrentProgram()</slot>
  <slot>ShowProgramView()</slot>
 </slots>
</ui>
